dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/json/DynamicSimulationResultSerializer.java

```diff
@@ -40,7 +40,6 @@ public void serialize(DynamicSimulationResult result, JsonGenerator jsonGenerato
         jsonGenerator.writeStartObject();
         jsonGenerator.writeStringField("version", VERSION);
         jsonGenerator.writeBooleanField("isOK", result.isOk());
-        jsonGenerator.writeStringField("logs", result.getLogs());
         jsonGenerator.writeEndObject();
```

Comment:
I don't understand this change: I would expect to serialize the curves and the timeline instead. Maybe it's better than a zip file.
@zamarrenolm @marcosmc what do you think?

Reply:
Done.